src/Uno.UWP/UI/Input/GestureRecognizer.Manipulation.cs

```diff
@@ -202,19 +255,40 @@ private ManipulationDelta GetDelta(Points from)
 				};
 			}
 
-			private bool IsSignificant(ManipulationDelta delta)
-				=> Math.Abs(delta.Translation.X) >= MinManipulationDeltaTranslateX
-				|| Math.Abs(delta.Translation.Y) >= MinManipulationDeltaTranslateY
-				|| delta.Rotation >= MinManipulationDeltaRotate // We used the ToDegreeNormalized, no need to check for negative angles
-				|| Math.Abs(delta.Expansion) >= MinManipulationDeltaExpansion;
+			private ManipulationDelta GetDelta(ManipulationDelta cumulative)
+			{
+				var deltaSum = _sumOfPublishedDelta;
+
+				var translateX = _isTranslateXEnabled ? cumulative.Translation.X - deltaSum.Translation.X : 0;
```

Comment:
Are those cases tested ?

Reply:
Yes https://github.com/unoplatform/uno/blob/master/src/Uno.UI.Tests/Windows_UI_Input/Given_GestureRecognizer.cs#L297 If it what you are asking about